script/openqa-clone-job

```diff
@@ -149,7 +153,6 @@ https://openqa.opensuse.org 123456
 
 Will create 50 clones of the same job.
 
-
```

Comment:
Possibly unrelated change. I suggest you just keep it as is or create a separate commit (not separate PR) stating what this fixes. Did you see a problem in the generated help text?

Reply:
Will add it as separate commit though.